src/components/MovieDetails.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+
+import '../components-styles/MovieDetails.scss';
+
+const MovieDetails = ({poster_path, title, vote_average, genres, release_date, runtime, overview}) => {
```

Comment:
What is runtime?

Reply:
> What is runtime?

runtime is the movie "length" in minutes, this value comes with response from back-end